src/components/tiles/map-tile/floorplan.tsx

```diff
@@ -0,0 +1,76 @@
+'use client';
+import clsx from 'clsx';
+import { useState } from 'react';
+import { MdLocationPin } from 'react-icons/md';
+
+import { Map } from './map';
+
+const map = [
+  'Regisztrációs pult',
+  'Bosch',
+  'Nokia',
+  'ICF Tech',
+  'Sicontact',
+  'Silicon Labs',
+  'Interactive Brokers',
+  'Paks II.',
+  'Barré',
+  'Nova',
+  'KUKA',
+  'OTP',
+  'Knorr-Bremse',
+  'SCHIS',
+  'Paripa',
+  'BME Suborbitals',
+  'BME Formula Racing Team',
+  'Schdesign',
+  'Kir-dev',
+  'SEM',
+  'HA5KFU',
+  'LEGO',
+];
+
+export function Floorplan() {
+  const [active, setActive] = useState(0);
```

Comment:
I think it's better if nothing is active by default

Reply:
my sugg: `= useState<number>(-1)`